ee/docker-ee/ubuntu.md

```diff
@@ -23,42 +23,41 @@ Docker Engine - Community users should go to
 [Get Docker Engine - Community for Ubuntu](/install/linux/docker-ce/ubuntu.md)
 **instead of this topic**.
 
-To install Docker Engine - Enterprise, you need to know the Docker Engine -
-Enterprise repository URL associated with your trial or subscription. These
+To install Docker Engine - Enterprise, you need to know the Docker Engine - Enterprise repository URL associated with your trial or subscription. These
 instructions work for Docker on Ubuntu and for Docker on Linux, which includes
 access to Docker Engine - Enterprise for all Linux distributions. To get this
 information, do the following:
 
-- Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content).
-- Each subscription or trial you have access to is listed. Click the **Setup**
+1. Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content).
+2. Each subscription or trial you have access to is listed. Click the **Setup**
   button for **Docker Enterprise Edition for Ubuntu**.
-- Copy the URL from the field labeled
+3. Copy the URL from the field labeled
   **Copy and paste this URL to download your Edition**.
 
 Use this URL when you see the placeholder text `<DOCKER-EE-URL>`.
 
-To learn more about Docker Enterprise, see
+To learn more about Docker Engine - Enterprise, see
 [Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
 
 ### OS requirements
 
 To learn more about software requirements and supported storage drivers,
 check the [compatibility matrix](https://success.docker.com/article/compatibility-matrix).
 
-> **Note:**
-> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.06.xx only.
+> **Note**
+> 
+> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.04.xx only.
 
 ### Uninstall old versions
 
 Older versions of Docker were called `docker` or `docker-engine`. In addition,
```

Comment:
Change the first sentence to read:, "Use the `sudo apt-get remove` command to uninstall older versions of Docker Engine - Enterprise (called `docker` or `docker-engine`).